x-pack/plugins/infra/public/alerting/metric_threshold/rule_data_formatters.ts

```diff
@@ -7,10 +7,10 @@
 
 import { ALERT_REASON } from '@kbn/rule-data-utils';
 import { ObservabilityRuleTypeFormatter } from '../../../../observability/public';
-
+import { LINK_TO_METRICS_EXPLORER } from '../../../common/alerting/metrics';
 export const formatReason: ObservabilityRuleTypeFormatter = ({ fields }) => {
   const reason = fields[ALERT_REASON] ?? '-';
-  const link = '/app/metrics/explorer'; // TODO https://github.com/elastic/kibana/issues/106497 & https://github.com/elastic/kibana/issues/106958
+  const link = LINK_TO_METRICS_EXPLORER; // TODO https://github.com/elastic/kibana/issues/106497 & https://github.com/elastic/kibana/issues/106958
```

Comment:
Do I understand the AC correctly that this should be read from the alert data instead?

Reply:
AC was not updated/corrected. It's fixed now.